### ISSUE TITLE
Dependency of generated item to its layout never unmet

### ISSUE THREAD

**Issue by GeekyUX** [nanoc/nanoc#891]
Encountered in Nanoc 4.2.1

Steps to reproduce:

1. Create a new site
2. Add the preprocess rule:

  ~~~ ruby
  preprocess do
    items.create('', {}, '/hello.html')
  end
  ~~~

3. Compile; This creates output/index.html and output/hello/index.html

4. Modify layouts/default.html in some way (I've been adding exclamation marks to `<title>`)

5. Recompile

Expected outcome: Both output/index.html and output/hello/index.html should be updated.
Actual outcome: Only output/index.html is updated.

**Comment by GeekyUX** [comment]
Note: I believe this is related to issue #885

**Comment by ddfreyne** [comment]
Oh yikes—definitely a bug.

**Comment by ddfreyne** [comment]
Fix is in #893.

**Comment by GeekyUX** [comment]
Thanks, Denis! That was quick! :+1:
